pkg/R/utils.R

```diff
@@ -419,3 +419,21 @@ cleanClosure <- function(func, checkedFuncs = new.env()) {
   }
   func
 }
+
+# Merge another StatCounter into this one, adding up the internal statistics.
```

Comment:
We could use some comments here as to what structure `x` and `y` have (i.e. what is x[1] and x[2]).  Also a light-weight method to get OOP is to use named lists like `list(sum=x1, sumsq=x2)` etc.